pylint/checkers/imports.py

```diff
@@ -238,6 +238,9 @@ def _make_graph(filename, dep_info, sect, gtype):
               'module',
               'wrong-import-position',
               'Used when code and imports are mixed'),
+    'C0414': ('Useless import alias:%s',
+              'useless-import-alias',
+              'Used when an import alias does not rename the import'),
```

Comment:
Can you add a more descriptive help message? It might not be obvious from the help message where the problem lies.